api/helpers/function_invoke_helper.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const aws = require('aws-sdk');
+const lambda = new aws.Lambda();
+
+module.exports.invokeFunction = (params, context) => {
+    lambda.invoke(params, function(err, data) {
+        if (err) {
+            context.fail(err);
```

Comment:
context.succeed & context.fail is deprecated (Node.js runtime v0.10.42). Use callback instead (for Node.js runtime v4.3 or v6.10)